gdnative-core/src/variant.rs

```diff
@@ -153,6 +153,35 @@ impl VariantType {
     }
 }
 
+#[repr(u32)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
+pub enum CallError {
+    InvalidMethod = sys::godot_variant_call_error_error_GODOT_CALL_ERROR_CALL_ERROR_INVALID_METHOD,
```

Comment:
Does this work properly on Windows? I remember that some enums representing Godot enums had to cast to `u32` because on Windows the enum values are `i32` instead.

Reply:
Totally missed that. Added casts to `u32`. Thanks!